components/GitHubAccelerator.js

```diff
@@ -1,5 +1,5 @@
 import Image from "next/image";
-import { FaGithub, FaMoneyBillAlt } from "react-icons/fa";
+import { FaMoneyBillAlt, FaGithub } from "react-icons/fa";
```

Comment:
Removing unnecessary change

```suggestion
import { FaGithub, FaMoneyBillAlt } from "react-icons/fa";
```